src/Compute/Compute/help/New-AzDiskEncryptionSetConfig.md

```diff
@@ -108,6 +109,21 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -RotationToLatestKeyVersionEnabled
+{{ Fill RotationToLatestKeyVersionEnabled Description }}
```

Comment:
Please change description for parameter help message.

Reply:
Added